waspc/examples/todoApp/src/server/auth/email.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  GetPasswordResetEmailContentFn,
+  GetVerificationEmailContentFn,
+} from '@wasp/types'
+
+export const getPasswordResetEmailContent: GetPasswordResetEmailContentFn = ({
```

Comment:
User defined e-mail templates

Reply:
In the future maybe we can use some sort of templating for writing the text/html strings more ergonomically. Maybe something simple like loading Handlebars/Mustache templats, or maybe even rendering React components to strings (https://legacy.reactjs.org/docs/react-dom-server.html)?

Reply:
I agree 100%, we should consider templating for e-mails in some future iteration. Then this will benefit the users when using e-mail auth. There is thing interesting library for creating e-mails with React: https://react.email/docs/utilities/render